week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,12 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? Returns a random number between 0 and i (excluded ) then multiply it to 10
+// - How many parameters does the function take? 1
+// - What is the function name? getNumber
+// - Where is the function called? with what parameters? at five .
```

Comment:
This is correct, but when we are talking about a position in a source file we usually refer to the line number. So we would say something like "The function is called at line 5".